utils/utils.go

```diff
@@ -215,3 +128,66 @@ func defaultGOPATH() string {
 	}
 	return ""
 }
+
+func getImportPath(name string, key string) (string, error) {
```

Comment:
This is really great, I should have removed the duplicate code from the beginning 👍